### ISSUE TITLE
Change deprecated `importlib.resources.path` call

### ISSUE THREAD

**Issue by dylanhmorris** [CDCgov/forecasttools-py#45]
```forecasttools/__init__.py:31
  /home/runner/work/forecasttools-py/forecasttools-py/forecasttools/__init__.py:31: DeprecationWarning: path is deprecated. Use files() instead. Refer to https://importlib-resources.readthedocs.io/en/latest/using.html#migrating-from-legacy for migration advice.
    with importlib.resources.path(
```

**Comment by AFg6K7h4fhy2** [comment]
Worth noting that this issue is also related to #42 ; the two can likely be covered in the same PR.

**Comment by AFg6K7h4fhy2** [comment]
Closed via #48 .
